android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/LogTest.java

```diff
@@ -488,23 +457,24 @@ public void testLogFileConfigurationConstructors() {
     }
 
     @Test
-    public void testEditReadOnlyLogFileConfiguration() {
-        final File path = new File(
-                context.getCacheDir().getAbsolutePath(),
-                "testEditReadOnlyLogFileConfiguration"
-        );
+    public void testEditReadOnlyLogFileConfiguration() throws Exception {
+        final File path = tempFolder.newFolder();
         final String logDirectory = emptyDirectory(path.getAbsolutePath());
         LogFileConfiguration config = new LogFileConfiguration(logDirectory);
-        Database.log.getFile().setConfig(config);
 
```

Comment:
These tests do not, in fact, throw `Exception`.  I would prefer that the signatures be changed to reflect the exact exception that is thrown, if any.

Reply:
i thought this exception which is thrown is with creating the tempFolder. I can update with exact exception.